shell/platform/tizen/channels/platform_channel.cc

```diff
@@ -24,6 +35,94 @@ PlatformChannel::PlatformChannel(flutter::BinaryMessenger* messenger)
 
 PlatformChannel::~PlatformChannel() {}
 
+#if defined(MOBILE) || defined(WEARABLE)
+namespace {
+
+class FeedbackManager {
+ public:
+  enum class ResultCode {
+    OK,
```

Comment:
https://google.github.io/styleguide/cppguide.html#Enumerator_Names
Could you please apply the Google Coding Style Guide if possible?